### PR TITLE
Components: Calculator layout

### PR DESCRIPTION
**Pull request by kemigabocatherine**
### Changes made

- Added new directory components containing new files with calculator code
- Added a new file calculator .js containing react component for  my calculator
- Added a calculator.css file for adding styles to the calculator

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
```
Comment (Ghiftee) [inline src/App.js]: - [ ] Linters are useful for ensuring we write clean code so please do not disable them.

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable no-unused-vars */
```
Comment (Ghiftee) [inline src/components/calculator.js]: - [ ] Linters are useful for ensuring we write clean code so please do not disable them.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @kemigabocatherine

Good job working on the project so far! 
✔️ Your app displays correctly
✔️ Your components are class based
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._
- [ ] There are still some CRA boilerplate located in your public folder, kindly delete them.
- [ ] In your README, please remove the acknowledgment section or fill it out.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @kemigabocatherine  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done working on the requested changes. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**